src/Landing/containers/LatestArticles.jsx

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prop-types */
+import React, { Component } from 'react';
+import { Container } from 'react-bootstrap';
+import { connect } from 'react-redux';
+import Article from '../../commons/components/Article';
+import '../CSS/RecommendedArticles.scss';
+import ActionButton from '../components/ActionButton';
+import fetchLatestArticles from '../reduxx/actions/LatestArticles';
+
+class LatestArticles extends Component {
+  componentWillMount() {
+    this.props.fetchLatestArticles();
+  }
+
+  render() {
+    this.articleItems = this.props.latest_articles.map(article => (
```

Comment:
Use destructuring of props an example is instead of 
`this.props.latest_articles.map`
first do
`const { latest_articles } = this.props`
then
`latest_articles.map(articles => ...`

Reply:
I am done working on this. thanks.